app/controllers/contratos_controller.rb

```diff
@@ -5,18 +5,25 @@ def index
        render json:@contratos
   end
 
+  def values_aleatory ()
+    return @value_true*(1+rand(30))/(1+(rand(30)))
```

Comment:
Redundant `return` detected.<br>Surrounding space missing for operator `/`.<br>Surrounding space missing for operator `*`.<br>Surrounding space missing for operator `+`.